tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -124,6 +124,41 @@ describe(__filename, () => {
       expect(root.find(AddonReviewCard).at(0)).toHaveProp('review', null);
     });
 
+    it('shows 1:1 placeholders for reviews belonging to an addon', () => {
+      const numberOfPlaceholders = 10;
+      const reviews = Array(numberOfPlaceholders).fill({
+        ...fakeReview,
+        id: 1,
+        score: 1,
+      });
+      _setAddonReviews({ reviews });
+
+      const root = render();
+
+      // Make sure a 1:1 ratio of reviews to placeholders is rendered.
+      expect(root.find(AddonReviewCard)).toHaveLength(numberOfPlaceholders);
+      // Do a sanity check on the first placeholder;
+      expect(root.find(AddonReviewCard).at(0)).toHaveProp('addon', null);
+      expect(root.find(AddonReviewCard).at(0)).toHaveProp('review', null);
```

Comment:
This is closer to what we want but still actually renders each review instead of the placeholder... 😕

I've skimmed through the other tests and didn't see the logic that we're looking for. Essentially, we will need a valid `addon` that is not null, has multiple reviews and figure out a way to not actually load each review (_hopefully that makes sense_).

It's been a long day and my brain is toast.. I'll take a look at it again tomorrow but any suggestions/insights would be awesome @willdurand 😃

Reply:
This is exactly what I was thinking, thank you!